backend/api/Startup.cs

```diff
@@ -95,21 +99,21 @@ public void ConfigureServices(IServiceCollection services)
                             {
                                 return Task.CompletedTask;
                             },
-                        OnAuthenticationFailed = context =>
-                        {
-                            context.NoResult();
-                            context.Response.StatusCode = 500;
-                            context.Response.ContentType = "text/plain";
-                            if (Environment.IsDevelopment())
+                            OnAuthenticationFailed = context =>
+                            {
+                                context.NoResult();
+                                context.Response.StatusCode = 500;
+                                context.Response.ContentType = "text/plain";
+                                if (Environment.IsDevelopment())
+                                {
+                                    return context.Response.WriteAsync(context.Exception.ToString());
+                                }
+                                return context.Response.WriteAsync("An error occurred processing your authentication.");
+                            },
+                            OnForbidden = context =>
```

Comment:
I understand that this is nobody's fault but the last few PRs we seem to be alternating between different spacing here depending on who submits a PR. I'm indifferent to spacing changes like these but let's please agree on a standard that works for everybody so we don't see this all the time.

Reply:
This is a vscode issue.  If we can get it to standardize that would be great.